WordPress/gutenberg
# [RNMobile] Correct displaying full-width Buttons block

## Description


Gutenberg-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3077

That PR introduces the fix for wrongly displayed full-width `Buttons`.

## How has this been tested?

1. Do sanity testing for Buttons
2. Add `Buttons` block
3. Fill it with some characters
4. Change the alignment to wide width
5. Type some characters again and take the whole available space
6. Rotate the device
7. Observe if buttons fit in the screen
8. Change the alignment to full width
9. Again observe if buttons fit in the screen





## Screenshots 

* iPhone 11 portrait mode (14.3)

default | wide | full
--- | --- | ---
<img width="527" alt="Screenshot 2021-01-27 at 13 59 49" src="https://user-images.githubusercontent.com/22746080/106124892-e06ba000-615b-11eb-9161-dd87b67468d9.png"> | <img width="527" alt="Screenshot 2021-01-27 at 13 59 46" src="https://user-images.githubusercontent.com/22746080/106125078-04c77c80-615c-11eb-93a1-127872d4c7a3.png"> | <img width="527" alt="Screenshot 2021-01-27 at 13 59 41" src="https://user-images.githubusercontent.com/22746080/106125251-380a0b80-615c-11eb-91b3-8d836020e245.png">

* iPhone 11 landscape mode (14.3)

default | wide | full
--- | --- | ---
<img width="922" alt="Screenshot 2021-01-27 at 13 59 35" src="https://user-images.githubusercontent.com/22746080/106125951-11000980-615d-11eb-8dbe-0d55033c0adc.png"> | <img width="922" alt="Screenshot 2021-01-27 at 13 59 30" src="https://user-images.githubusercontent.com/22746080/106125965-152c2700-615d-11eb-9f28-66b2eb9b266a.png"> | <img width="922" alt="Screenshot 2021-01-27 at 13 59 22" src="https://user-images.githubusercontent.com/22746080/106125976-19584480-615d-11eb-99d7-5b66dabda5e4.png">

* iPad

portrait | landscape
--- | ---
<img width="679" alt="Screenshot 2021-01-27 at 13 48 36" src="https://user-images.githubusercontent.com/22746080/106127093-7b657980-615e-11eb-91c8-4e37dff5f760.png"> | <img width="850" alt="Screenshot 2021-01-27 at 13 58 37" src="https://user-images.githubusercontent.com/22746080/106127104-7e606a00-615e-11eb-8e01-5df0444c7f3e.png">

* android portrait

default | wide | full
--- | --- | ---
<img width="428" alt="Screenshot 2021-01-28 at 11 54 31" src="https://user-images.githubusercontent.com/22746080/106128661-da77be00-615f-11eb-9e15-7eabbf631e22.png"> | <img width="428" alt="Screenshot 2021-01-28 at 11 54 39" src="https://user-images.githubusercontent.com/22746080/106128670-dc418180-615f-11eb-8299-0b848877d3c4.png"> | <img width="428" alt="Screenshot 2021-01-28 at 11 54 45" src="https://user-images.githubusercontent.com/22746080/106128675-de0b4500-615f-11eb-9558-2e1f0284a449.png">



## Types of changes





## Checklist:
